manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -394,11 +394,13 @@ properties:
n/m3OYELEGt7cVnaFjQO5hVzJuBPk/ECPk1E/23i2ZlX0XetkwJ4NDQvgKtEmN2Uey7yLOyAAAYAWJOZH69yPfzk+9K81H1xHN8HtSi1PTDG7NZAQIV8s67Jq7cdENFpVftERFOMMd/PBjNl5vnOuWDT5ApE3p1icovzgPn+G9FMXve1mHlUM+37Y4ZDScyEHAH8ILQv8KLm4gCx70ITc1ZOLSBVEcAAAEqpbeM4bhRstqjMRsRGUfDr8C5jU5toKyGil7XWB0An5outCODULirsNqS87DokgR9cVvvvPYeW73juX9sFDqq7qOw7vFZPozb2GVZ/3D4NjttraM7vhzVuJ7VokNvP3g6C5fkEEc3Oe1mkxOgcZp5ORK/4l+2/iehGZh5GRBcYY/5ojDlJKXW01vrnWuuDtdb7GWP2jKLoB77sFEXRDr7s7P+2qzFmb631AVrrQ7TWv1JKHWeMOdVaey4RXczMVxHRrd5X6Dkiet2bjcxrJJi7I+wtwWyEiOcS0ctVXub+ty/m5Q2O4/j6RosKEJbntRFdzrmDuJbPd24T/Z5PRKMR8QhoUxqJRjDz41UWXYhoRDPtOOdObDAWc3zAkFbYGBH/REQvVf3w+9+/RURDGvnHVeBbXV1dZeNYJcpwHszMY8quV1dXV3dApl6PCm2M2TA7Dr0YnGs1Zh7dYEU9Mcbk5kx2zu0fx/HzAffRPD9ZXAsAwFr7jdjvPFcVn9babzDzlLLdoxL/vC5mfilz7Q+qNGqLYR8p/vWAcVjIzKOqRCf2rJ/z/m2U8q8Q59xx6XuNa0HMggQwIu6d+U6+DuHCY7N0u11dXYnWerNmziGA5bsX+8ruE2Z+1Vp7WPeB3r+z+zcvQuPgSMv4aLxPh7xL/bk/b639aYfOexE+4NzYgH4tIKJ/gt8NU0ptnV5kstYe2uGurmKMOZZrLgULQ8eRa7v3Vyultm6x/cha+/M4jieELD5zbXd4hLX22yGVO+d+m3neHoc2fmOstWcXfDdHNlMfMz+afkaNMadUrGI1RLwgjuNpAc/eAiK6rQXrq42Z+Woi+jDk2nnNcRosqUGwpPRdaZMAVsx8LyJOQcR7EXGotfZMrfVhxpjd/KR5faiJrW7fjr5IQxJBLaDVMgCwitZ6E6XU9lrrwUqpYxHxPES8DhH/y8zvEVEzuc36gkFRFH3HOXcGEd2CiOOY+V2/ezmTmacj4muI+AAi/tUYsysUTGCiKNpRKXWCUuroTDlGKXVE0XFNMAAANtBaH0pElyDig4j4CjNP832eybUFk9cR8VFEHKqUOtLnfO61HTul1HYF45FXjgeAIr+zUqy13/KLQu/4838LEa9tk+jspksp9Q1vDni9N/V6O47j6cw8M47j6UT0LhGNJaIRiHiWD7xVaVe7hLWUUr/x91TeOJ4QEmgkgM19O42u2W+iKDoI+iatwnJa619orQ/zC3e/hjb73pUwCBGPYuY7EfFF/4540S9QjtJaH9joWK31YCK6DBFHI+Ir/v0/johuc86dlCd0tNZbENFtTfqOf9w5dzwRjUTECb69VxFxHCI+RET/KklNpP27/gg/3kcCwKeb6EealbTWBxHRED8OL/t+jSeikc65M1qYrC/r743u8ito/jncTCl1VPd5e7+5oCjeALBu6tgj/C5KqCvGin5B+/A2nEMoGxhjjieim4lorL+vXyGiJ5h5mHPuR3l9sNYeQkQPej/UEKxS6qvOuZOI6F9E9KR/Dl5DxBcQ8T5EvMAYszu0kJO+CWJjzF5E9Hf/7ex+Nif4e/K0gjSBmyLiTa1a31TEKqW+5KNED0fE//ixm4KIU/z1G0VEF/j3UUjcgCrExphdEfH/+ev1gr9+LxHRY4h4pbX2KGPMxhXr/byfr3Q/b7tBe+e86/pn8TCt9a98O0dBYIyQLFEU7Zrub7P1AMA61trDvEB9yr+fX/PziVv8d6FS+rgGbGiMOZqZr/VtLbrPEXEUIv7Fz3VD/MNbQwRw/yxtEsAANWEUw5KRX7NVFNQemj6P/twkDmrXYzWo+cesDLUP8JKVO60nA6FmGrYCAHwcav1eCWr9bpgs/SPIclAbg/8r+2EbMFAzZ1zZt/kxqN07nTIrF5ZMCGrvb4IKfuYeBbVnN4beu2/Qt8e+zSXh3dYX4yA0xsHi+yRk4bSV+0j7drqgdn8uCXOh9D3ZH+Yz3ZsUMfT+czQAer5X+so15qOCgd57H6bb6v37XARw/yxtFMCCIAiCIAiCIAhLByKA+2cRASwIgiAIgiAIglAREcD9s4gAFgRBEARBEARBqIgI4P5ZRAALgiAIgiAIgiBURARw/ywigAVBEARBEARBECoiArh/FhHAgiAIgiAIgiAIFVlz61XgRzfN6nNBJ6W6AH7klV37+vYRBEEQBEEQBEHoP6z6uRVg8PX3wu6XPyWlH5XvXvL00IcmhuZ9FwRBEARBEARBEAAArr322uikf/9bSelfBZaMfO2CIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCIAiCICxl/H9l333eOJxP0AAAAABJRU5ErkJggg==
```

Comment:
I didn't notice until reviewing #1106 but it belongs to this commit..

The image, particularly the "beta" text, looks fuzzy to me. I don't *think* it's my eyes or screen. Am I imagining it? Is there anything that can be done about it?

![screen shot 2017-11-17 at 11 21 08](https://user-images.githubusercontent.com/260438/32945111-12ab8ee2-cb8a-11e7-81ff-891c51a742c1.png)

Reply:
That *does* look fuzzy, and suggests that the underlying PNG image is too small. I'll look into it and maybe ask Stephen for a larger version. Thanks Dan 💯

Reply:
It looks okay to me with the default UAA styling, even when blown up. I'll deploy the second PR and see if it looks distorted there.

<img width="909" alt="screenshot 2017-11-17 13 52 53" src="https://user-images.githubusercontent.com/163111/32950583-ba2e8368-cb9e-11e7-9ac4-4dca1f275cf6.png">

Reply:
@dcarley Oddly, on a similar scale it looks okay for me.

<img width="495" alt="screenshot 2017-11-17 15 48 09" src="https://user-images.githubusercontent.com/163111/32955756-ce52e41e-cbae-11e7-8318-a5d111e6acc7.png">

I don't know what to do about this. That it appeared in a screenshot shows it isn't your eyes or your screen. It can't be a simple caching issue as the image is an inline data URI in the HTML file, and you haven't worked on this before. The image is sharp and 960px wide. Thoughts?

Reply:
Wait. You're on the invite page…

Reply:
I get the same on the login page too:

![screen shot 2017-11-17 at 15 55 57](https://user-images.githubusercontent.com/260438/32956139-e27cf49c-cbaf-11e7-88d6-a5e77448e9e0.png)

I checked if it was any different in Safari, but I think it's uncovered a new issue whereby it's not styled at all. Any idea why?

![screen shot 2017-11-17 at 15 59 20](https://user-images.githubusercontent.com/260438/32956408-972151fe-cbb0-11e7-998b-abe28104e633.png)

Reply:
Safari is probably blocking the app's domain because its certificate doesn't validate. If you [visit the `paas-uaa-assets` app](https://paas-assets-app.dcarley.dev.cloudpipelineapps.digital/) and accept its certificate as being invalid, then hopefully refreshing the login page will show the assets.

Reply:
If this can use external assets, does this need to be an inline data-uri, or could it also point at an external asset?

Reply:
As part of making it look good on non-retina screens, we've switched to serving the image from the assets app.